CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ### Features
 - Avoid error when missing column in YAML description ([#4151](https://github.com/dbt-labs/dbt-core/issues/4151), [#4285](https://github.com/dbt-labs/dbt-core/pull/4285))
 
+### Fixes
+- Allow specifying default in Jinja config.get with default keyword ([#4273](https://github.com/dbt-labs/dbt-core/issues/4273), [#4297](https://github.com/dbt-labs/dbt-core/pull/4297))
```

Comment:
Note to self to call out that this is technically a breaking change for anyone using `config.get(arg1, arg2)`, and passing a validation function to `arg2`. I think that's a pretty small (or nonexistent) set of people